hibike/devices/BatteryBuzzer/safety.cpp

```diff
@@ -0,0 +1,79 @@
+#include "safety.h"
+
+//has code that determines if the battery is safe or not, and takes appropriate action.
+//this code is regularly called (currently every 250ms)
+bool unsafe_status = false;
+
+float min_cell = 3.3;
+float end_undervolt = 3.6; //I have to exceed this value to remove my undervolt condition.  (Aka if loading conditions momentarily spiked the battery...)
+bool under_volt = false; //if i undervolt, i'll have to keep track if i'm no longer in an undervolt condition to avoid momentary beeps.
+
+
+float max_cell = 4.4;
+
+float d_cell = .3; //max voltage difference between cells.
+float end_d_cell = 0.1; //imbalance must be less than this before i'm happy again.
+bool imbalance = false;
+
+
+
+
+void handle_safety()
+{
+  unsafe_status = is_unsafe(); //Currently, just does basic sensing.  Should get updated.
+  buzz(unsafe_status);
```

Comment:
unsafe_status is a kinda wierdly named bool.

is_safe, unsafe, etc, makes more sense.